CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ All notable changes to this project will be documented in this file.
     - Added line counter to ticker log
     - Refactored task manager so it becomes easier to use
     - Global Client Error Handler: Grabs any errant / uncaught errors and displays a modal
+    - The ability to create a github issue from the interface after encountering an error.
```

Comment:
Maybe add stuff about reworking OctoPrint client. There's changes related to that which currently seem broken

Reply:
Well it's not finished, that's why. You left me hanging on my question regarding it, but I'll just sort it regardless.

Reply:
I also need the OP client working to test further the OP changes...